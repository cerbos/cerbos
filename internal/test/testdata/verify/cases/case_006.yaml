description: Invalid test suite with missing fixtures
