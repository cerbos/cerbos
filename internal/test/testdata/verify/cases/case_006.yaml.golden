{
  "suites": [
    {
      "file": "suite_test.yaml",
      "name": "TestSuite",
      "summary": {
        "overallResult": "RESULT_ERRORED"
      },
      "error": "failed to load test fixtures from testdata: invalid TestFixture_Principals.Principals[invalid]: embedded message failed validation | caused by: invalid Principal.Id: value length must be at least 1 runes"
    }
  ],
  "summary": {
    "overallResult": "RESULT_ERRORED"
  }
}